templates/firefox-ios.xliff

```diff
@@ -4405,6 +4437,17 @@ Private Tab</source>
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/en.lproj/TabToolbar.strings" datatype="plaintext" source-language="en" target-language="en">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.0.1" build-num="15A507"/>
+    </header>
+    <body>
+      <trans-unit id="TabToolbar.Accessibility.DataClearance.v122" xml:space="preserve">
```

Comment:
Also checking-in as this string is marked as a v122 string

Reply:
Yes, this requirement came out late, but this string is part of the felt privacy feature going out in v122